arch/platform/cooja/dev/cooja-radio.c

```diff
@@ -78,19 +79,32 @@ enum {
 };
 
 /* COOJA */
-char simReceiving = 0;
-char simInDataBuffer[COOJA_RADIO_BUFSIZE];
-int simInSize = 0;
+int simReceiving = 0;
```

Comment:
Changing these types requires updates in the Cooja module as well, so the build system version must be bumped. Does the new code require these types to change, or is it a cleanup?

Reply:
This is probably why the builds fail. I adapted Cooja to issue those interrupts (https://github.com/kkrentz/cooja/).